index.js

```diff
@@ -2,15 +2,15 @@ const createRvClient = (
     orgId,
     publicKey,
     privateKey,
-    apiBaseUrl = 'https://dq-api.plauti.io',
+    apiBaseUrl = 'https://jarvis-dev.plauti.io/v3', // TODO: Update to prod url once Jarvis is released 
```

Comment:
Think we should already have prod as base URL, just not release the latest NPM version with this change.

Reply:
Yes, it is in the TODO: once we know the url we update this before merging and publish new version